src/renderer/components/VideoController.tsx

```diff
@@ -0,0 +1,41 @@
+import { styled, Box } from '@mui/material';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import Forward10Icon from '@mui/icons-material/Forward10';
+import Replay10Icon from '@mui/icons-material/Replay10';
+import colors from '../colors';
+
+const VideoControllerBox = styled(Box)`
+  background: ${colors.grey[700]};
+  color: ${colors.grey[300]};
+  height: 62px;
+  width: 100vw;
+  margin: 0;
+  padding: 11px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  column-gap: 24px;
+`;
+
+const VideoController = () => {
+  return (
+    <VideoControllerBox>
+      <div
+        style={{
+          backgroundColor: colors.grey[600],
+          fontWeight: 'bold',
+          fontSize: '24px',
+          borderRadius: '5px',
+          padding: '0 19px',
+        }}
+      >
+        00:00:00
+      </div>
+      <Replay10Icon sx={{ fontSize: '36px' }} />
```

Comment:
I believe these will need to be [icon buttons](https://mui.com/material-ui/api/icon-button/) so they can have an `onClick` handler

Reply:
Could you also please add an IconButton with a pause icon? Maybe just test it and then comment it out. 
Team Farquaad can add the logic for switching between the two if you like.

Reply:
Done and done, ill push the commits soon